ci/svd2rust-regress/src/diff.rs

```diff
@@ -257,7 +257,7 @@ impl Diffing {
         Ok([baseline, current])
     }
 
-    fn get_source_and_command<'s>(&'s self) -> [Option<(Source, Command)>; 2] {
+    fn get_source_and_command<'s>(&'s self) -> [Option<(Source<'s>, Command<'s>)>; 2] {
         let split = |s: &'s str| -> (Source, Command) {
```

Comment:
of use function instead of closure
```suggestion
    fn get_source_and_command(&self) -> [Option<(Source, Command)>; 2] {
        fn split(s: &str) -> (Source, Command) {
```